docs/pages/examples.js

```diff
@@ -17,9 +17,11 @@ import AppropriateImage from '../components/appropriate-image';
 import imageConfig from '../img/dist/image.config.json'; // eslint-disable-line
 
 const meta = {
-    title: 'Mapbox GL JS Examples',
+    title: 'Examples',
     description: 'Code examples for Mapbox GL JS.',
-    pathname: '/examples'
+    pathname: '/examples',
+    contentType: 'example',
```

Comment:
Similar to the discussion related `/help/tutorials`... do we want to use an `example` contentType here on the landing page? This page doesn't really follow the `example` content type guidelines. 🤷‍♀️

Reply:
Good point! I'll remove 🙇